packages/number-field/stories/number-field.stories.ts

```diff
@@ -454,6 +448,31 @@ readOnly.args = {
     readonly: true,
     value: '15',
 };
+export const validationIcons = (args: StoryArgs): TemplateResult => {
+    return html`
+        <sp-field-label for="validation">
+            Invalid Number Field without Stepper
+        </sp-field-label>
+        <sp-number-field
+            id="validation"
+            ...=${spreadProps(args)}
+            @change=${args.onChange}
+        ></sp-number-field>
+        <sp-field-label for="validation">
+            Valid Number Field with Stepper
+        </sp-field-label>
+        <sp-number-field id="validation" valid></sp-number-field>
+        <sp-field-label for="validation">
+            Invalid Number Field with Stepper
+        </sp-field-label>
+        <sp-number-field id="validation" invalid></sp-number-field>
```

Comment:
This could be my lack of knowledge of web components showing, but does having multiple `sp-number-field`s with the same `id="validation"`, and multiple corresponding `sp-field-label`s with the same `for="validation"` pose any sort of accessibility issue?

Reply:
oh good catch i should update that in the story